includes/Data/Patterns.php

```diff
@@ -281,8 +263,11 @@ public static function get_theme_step_patterns_from_step( $step, $squash = false
 		foreach ( array_keys( $pattern_slugs ) as $pattern_slug ) {
 			if ( true === $pattern_slugs[ $pattern_slug ]['active'] ) {
 				$pattern_name = $active_theme . '/' . $pattern_slug;
+				$pattern      = $block_patterns_registry->get_registered( $pattern_name );
```

Comment:
This needs to go back inside the `if($block_patterns_registry->is_registered( $pattern_name ))` we cannot `get_registered` without first checking if it `is_registered`